includes/abstracts/abstract.llms.post.model.php

```diff
@@ -812,51 +813,6 @@ protected function get_creation_args( $args = null ) {
 		return apply_filters( "llms_{$this->model_post_type}_get_creation_args", $args, $this );
 	}
 
-	/**
-	 * Get media embeds
-	 *
-	 * @since 3.17.0
-	 * @since 3.17.5 Unknown.
-	 *
-	 * @param string $type Optional. Embed type [video|audio]. Default is 'video'.
-	 * @param string $prop Optional. Postmeta property name. Default is empty string.
-	 *                     If not supplied it will default to {$type}_embed.
-	 * @return string
-	 */
-	protected function get_embed( $type = 'video', $prop = '' ) {
```

Comment:
A 3rd party extending class can still call `get_embed()` even if it's in a trait instead of in this class. To the extending class, there is no difference. It can even override the method if it needs to.

Reply:
@pondermatic 
I don't think we understood each other on this. :D
We're removing the method `get_embed()` from the `LLMS_Post_Model` abstract class, moving it to the trait `LLMS_Trait_Audio_Video_Embed`, but `LLMS_Post_Model` doesn't use the trait `LLMS_Trait_Audio_Video_Embed`.
So if a 3rd party is for any reason extending `LLMS_Post_Model` and referring to `$this->get_embed();` what happens?

Reply:
Oh! Now I see. That's a very important point!

I think I need to move `get_embed()` back to the `LLMS_Post_Model` class. `LLMS_Trait_Audio_Video_Embed` will just define `get_audio()` and `get_video()`.